packages/frontend/component/src/ui/slider/slider.tsx

```diff
@@ -0,0 +1,52 @@
+import * as Sliders from '@radix-ui/react-slider';
+import { useRef } from 'react';
+
+import * as styles from './index.css';
+
+export interface SliderProps extends Sliders.SliderProps {
+  nodes?: number[]; // The values where the nodes should be placed
+}
+
+export const Slider = ({
+  value,
+  min,
+  max,
```

Comment:
```suggestion
  min = 0,
  max = 1,
```
Giving default values to simplify calculation below? (Not sure if `max` should be required)